### ISSUE TITLE
Notifications print unhelpful error on Windows 7

### ISSUE THREAD

**Issue by LigH-de** [hexchat/hexchat#2152]
After updating to v2.14.2 in Windows 7 SP1 Pro (German), the startup protocol reports:

```
 Lua version 1.3/5.1 loaded.
 Perl interface loaded
 Python interface loaded
 Bookmarks version 2 loaded.
 Update Checker plugin loaded
 Failed loading notification plugin: hcnotifications-winrt not found.
```

But the file .\plugins\hcnotifications-winrt.dll exists in my installation of the Windows x86-64 version (not in C:\Program Files - so not monitored by the UAC).

**Comment by TingPing** [comment]
`<LigH> Windows 7 Pro`

Windows 7 doesn't support notifications. The error should probably be hidden though.

**Comment by notsag2d** [comment]
Issue stops hexchat from connecting to servers. Any update on this?
(win7 sp1)

>  Checksum plugin loaded
>  Exec plugin loaded
>  FiSHLiM plugin loaded
>  Python interface loaded
>  Sysinfo plugin loaded
>  Failed loading notification plugin: hcnotifications-winrt not found.

**Comment by norskkriger** [comment]
bump... can we get a / command to disable this or something in the new release?

**Comment by tNbh2KonoPZ7IonJYdt1d4je1lnoOXNG** [comment]
You can suppress this error message by commenting out [line 189](https://github.com/hexchat/hexchat/blob/8f2429a2d2064ec7cb4e28669f42b37610244c6a/src/fe-gtk/plugin-notification.c#L189) of `src/fe-gtk/plugin-notification.c` and building from source.

If you just want a quick "fix" and have HexChat v2.14.3 x64 installed:
1. Open `C:\Program Files\HexChat\hexchat.exe` in a hex editor (such as [HxD](https://mh-nexus.de/en/hxd/)).
2. Go to offset `0x279D1`.
3. Change `0x74` to `0xEB`.
4. Save the file.

**Comment by peterbmckinley** [comment]
> 

This didn't work for me (HexChat v2.16.0 x64, Win7 64bit)

Offset 0x29CC5 doesn't appear to exist in the Windows 7 X64 version of Hexchat

**Comment by peterbmckinley** [comment]
> You can suppress this error message by commenting out [line 189](https://github.com/hexchat/hexchat/blob/8f2429a2d2064ec7cb4e28669f42b37610244c6a/src/fe-gtk/plugin-notification.c#L189) of `src/fe-gtk/plugin-notification.c` and building from source.
> 
> If you just want a quick "fix" and have HexChat v2.16.0 x64 installed:
> 
>     1. Open `C:\Program Files\HexChat\hexchat.exe` in a hex editor (such as [HxD](https://mh-nexus.de/en/hxd/)).
> 
>     2. Go to offset `0x29CC5`.
> 
>     3. Change `0x75` to `0x73`.
> 
>     4. Save the file.
> 
> 
> Edit: Updated instructions for v2.16.0

This didn't work for me (HexChat v2.16.0 x64, Win7 64bit)

Offset 0x29CC5 doesn't appear to exist in the Windows 7 X64 version of Hexchat

**Comment by tNbh2KonoPZ7IonJYdt1d4je1lnoOXNG** [comment]
> [...]
> This didn't work for me (HexChat v2.16.0 x64, Win7 64bit)
> 
> Offset 0x29CC5 doesn't appear to exist in the Windows 7 X64 version of Hexchat

The above patches were successfully tested on Windows 7 with the following `hexchat.exe` binaries installed by the Windows 7 x64 setups downloaded from the [official website](https://hexchat.github.io/downloads.html):

setup|size|CRC32|offset|patch
---|---|---|---|---
v2.14.3 ([download](https://dl.hexchat.net/hexchat/HexChat%202.14.3%20x64.exe))|912896|91426681|0x279D1|0x74 -> 0xEB
v2.16.0 ([download](https://dl.hexchat.net/hexchat/HexChat%202.16.0%20x64.exe))|927744|4BFFB764|0x29CC5|0x75 -> 0x73
v2.16.1 ([download](https://dl.hexchat.net/hexchat/HexChat%202.16.1%20x64.exe))|924672|390BD997|0x29925|0x75 -> 0x73

Please double-check your installation.
